extension/awsproxy/README.md

```diff
@@ -1,6 +1,10 @@
 # AWS Proxy
 
-##
+| Status                   |           |
+|--------------------------|-----------|
+| Stability                | unknown   |
```

Comment:
```suggestion
| Stability                | beta   |
```

@mxiamxia let me know if you think this should be `stable` instead.  AFAIK the configuration has been stable for a while and I'm not aware of any plans to change it.